app/models/user.rb

```diff
@@ -12,4 +13,18 @@ class User < ActiveRecord::Base
 
   has_many :reports,
            dependent: :destroy
+
+  def self.find_for_google_oauth2(access_token, signed_in_resource=nil)
```

Comment:
Unused method argument - `signed_in_resource`. If it's necessary, use `_` or `_signed_in_resource` as an argument name to indicate that it won't be used.<br>Surrounding space missing in default value assignment.